tests/OSPSuite.Presentation.Tests/Importer/Core/DataFormat/DataFormat_MetaData_CSpecs.cs

```diff
@@ -1,108 +1,98 @@
 
 namespace OSPSuite.Presentation.Importer.Core.DataFormat
 {
-   public abstract class ConcernforDataFormat_DataFormatHeadersWithUnits : ContextSpecification<DataFormatHeadersWithUnits>
+   public abstract class concern_for_DataFormatHeadersWithUnits : ContextSpecification<DataFormatHeadersWithUnits>
    {
       protected DataSheet _basicFormat;
       protected ColumnInfoCache _columnInfos;
       protected IReadOnlyList<MetaDataCategory> _metaDataCategories;
-      protected IDimension _fakedTimeDimension;
-      protected IDimension _fakedConcentrationDimension;
-      protected IDimension _fakedErrorDimension;
 
       protected override void Context()
       {
-         _fakedTimeDimension = A.Fake<IDimension>();
-         _fakedConcentrationDimension = A.Fake<IDimension>();
-         _fakedErrorDimension = A.Fake<IDimension>();
-
          _columnInfos = new ColumnInfoCache
          {
-            new ColumnInfo() { DisplayName = "Time", Name = "Time", IsMandatory = true },
-            new ColumnInfo() { DisplayName = "Concentration", Name = "Concentration", IsMandatory = true },
-            new ColumnInfo() { DisplayName = "Error", Name = "Error", IsMandatory = false, RelatedColumnOf = "Concentration" }
+            new ColumnInfo() {DisplayName = "Time", Name = "Time", IsMandatory = true},
+            new ColumnInfo() {DisplayName = "Concentration", Name = "Concentration", IsMandatory = true},
+            new ColumnInfo() {DisplayName = "Error", Name = "Error", IsMandatory = false, RelatedColumnOf = "Concentration"}
          };
 
-         _columnInfos["Time"].SupportedDimensions.Add(_fakedTimeDimension);
-         _columnInfos["Concentration"].SupportedDimensions.Add(_fakedConcentrationDimension);
-         _columnInfos["Error"].SupportedDimensions.Add(_fakedErrorDimension);
+         _columnInfos["Time"].SupportedDimensions.Add(DimensionFactoryForSpecs.TimeDimension);
+         _columnInfos["Concentration"].SupportedDimensions.Add(DimensionFactoryForSpecs.ConcentrationDimension);
+         _columnInfos["Error"].SupportedDimensions.Add(DimensionFactoryForSpecs.ConcentrationDimension);
          _metaDataCategories = new List<MetaDataCategory>()
          {
-            new MetaDataCategory() { Name = "Organ" },
-            new MetaDataCategory() { Name = "Compartment" },
-            new MetaDataCategory() { Name = "Species" },
-            new MetaDataCategory() { Name = "Dose" },
-            new MetaDataCategory() { Name = "Molecule" },
-            new MetaDataCategory() { Name = "Route" }
+            new MetaDataCategory() {Name = "Organ"},
+            new MetaDataCategory() {Name = "Compartment"},
+            new MetaDataCategory() {Name = "Species"},
+            new MetaDataCategory() {Name = "Dose"},
+            new MetaDataCategory() {Name = "Molecule"},
+            new MetaDataCategory() {Name = "Route"}
          };
 
-         A.CallTo(() => _fakedTimeDimension.HasUnit("min")).Returns(true);
```

Comment:
@georgeDaskalakis Another example why we should never fake dimension. Suddently other tests are failing because details of implementation has changed. In general, we should only fake presenter, view and services (last only if really required). Domain, as a rule of thumb, never